src/ui/public/courier/fetch/call_client.js

```diff
@@ -94,7 +94,9 @@ export default function CourierFetchCallClient(Private, Promise, es) {
         throw ABORTED;
       }
 
-      return (esPromise = es[strategy.clientMethod]({ body }));
+      const id = strategy.id;
+      const client = (id && ~id.indexOf('admin')) ? esAdmin : es;
```

Comment:
I think `id && id.includes('admin')` is a bit easier read. Not many people are familiar with using `~` this way (and even fewer understand what it actually does)